admin/services/check-form.service.js

```diff
@@ -126,6 +126,32 @@ const checkFormService = {
       const promises = modifiedCheckWindows.map(cw => { cw.save() })
       return Promise.all(promises)
     }
+  },
+  /**
+   * Append check windows name(s) and canDelete to check form object.
+   * @param checkWindows
+   * @returns {Promise.<void>}
+   */
+  checkWindowNames: (checkWindows) => {
+    let checkWindowsName = []
+    checkWindows.forEach(cw => {
+      checkWindowsName.push(' ' + cw.checkWindowName)
+    })
+    return checkWindowsName
+  },
+  /**
+   * Append check windows name(s) and canDelete to check form object.
+   * @param checkWindows
+   * @returns {Promise.<void>}
+   */
+  canDelete: (checkWindows) => {
```

Comment:
is the jsdoc description right here?

Reply:
@pris54 please update the jsdoc description